1.9/usage/pods/pods-cli.md

```diff
@@ -0,0 +1,49 @@
+---
+post_title: Pods CLI
+menu_order: 20
+---
+
+You can create and manage your pods via the DC/OS CLI. The following commands are available:
+
+* dcos marathon pod add [<pod-resource>]
+* dcos marathon pod list [--json]
+* dcos marathon pod remove [--force] <pod-id>
+* dcos marathon pod show <pod-id>
+* dcos marathon pod update [--force] <pod-id> [<properties>...]
```

Comment:
The stuff in angle brackets is getting stripped out in the rendered document because Markdown treats them as HTML tags.